CHANGES.md

```diff
@@ -4,6 +4,13 @@
 - Fixed a detection issue of the BLAS libraires packaged by conda-forge on Windows.
   https://github.com/joblib/threadpoolctl/pull/112
 
+- `threadpool_info` and `ThreadpoolController.limit` accept a new value for the `limits`
+  parameter: the string "sequential_blas_under_openmp". It should only be used for the
```

Comment:
```suggestion
- `threadpool_info` and `ThreadpoolController.limit` now accept the string
  "sequential_blas_under_openmp" for the `limits` parameter.
  It should only be used for the
```

Reply:
yeah I was not happy with my phrasing. Thanks